src/components/home/InstitutionDetails.vue

```diff
@@ -60,6 +60,18 @@
           ></v-autocomplete>
         </v-list-item-content>
       </v-list-item>
+      <v-list-item v-if="!institution.existing">
```

Comment:
Remove v-if. All institutions can be deleted.

Reply:
updated